client/.env.example

```
@@ -0,0 +1,5 @@
+# Client environment variables
+# Next.js will expose variables prefixed with NEXT_PUBLIC_ to the browser
+NEXT_PUBLIC_API_BASE_URL=http://localhost:5000
+# Admin list for client-side admin gating (comma-separated)
+NEXT_PUBLIC_ADMIN_EMAILS=john@example.com
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)